theia-extensions/viewer-prototype/src/browser/trace-explorer/trace-explorer-sub-widgets/theia-trace-explorer-placeholder-widget.tsx

```diff
@@ -23,10 +28,17 @@ export class TraceExplorerPlaceholderWidget extends ReactWidget {
         this.update();
     }
 
+    dispose(): void {
+        super.dispose();
+    }
+
     render(): React.ReactNode {
-        const { loading } = this.state;
+        const { loading, serverStatus, tracesOpened } = this.state;
         return (
             <ReactExplorerPlaceholderWidget
+                tracesOpen={tracesOpened}
+                serverOn={serverStatus}
+                handleStartServer={this.handleStartServer}
```

Comment:
By the way, when porting this change to the `vscode-trace-extension`, the start server command is only available if `vscode-trace-server` is installed, which is not always the case. So, starting the trace should be handled there using the start/stop vscode API that is provided by the `vscode-trace-extension` and which extensions can contribute the start/stop functionality. The `vscode-trace-server` is contributing to that API.